src/containers/App/App.scss

```diff
@@ -93,9 +93,14 @@ body,
 .ydb-paginated-table__row,
 .ydb-tree-view__item {
     &:hover {
```

Comment:
Since you add padding only here, you should also add padding for focused state. Now if I select button with Tab, it won't have right padding

Reply:
I refactored styles. Wrapper is only needed on hover. If buttons are focused from keyboard, they should be shown separately.